roman_numeral/roman_numerals.py

```diff
@@ -0,0 +1,52 @@
+"""Given a roman numeral, find the integer equivalent"""
+
+# roman_numerals = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IV", "X"]
+
+
+number = 15
+
+roman_numeral_values = {1: "I", 5: "V", 10: "X"}
+
+
+def conversion(number):
+    """Convert an Arabic number to its roman numeral equivalent"""
+
+    if number == 0:
+        return ""
+
+    else:
+        convert = []
+        reduced_num = number
+
+        # First check: see if number is in dictionary. If so, return it, if not get the largest value from the dictionary
+        if number % 5 == 0:
+            try:
+                convert.append(roman_numeral_values[number])
+                reduced_num -= number
+            except KeyError:
+                convert.append(roman_numeral_values[10])
+                reduced_num -= 10
+
+            return "".join(convert) + conversion(reduced_num)
```

Comment:
Is it necessary to have `convert` as a list? It seems a string would be more than fine because you're ultimately converting it to a string on line 30 with `join` anyway.